lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -71,13 +76,33 @@ define((require, exports, module) ->
             left: leftNearest
             right: rightNearest
             mismatch: true
-        if result.left and result.right
-            expectedRightBracket = session.$brackets[session.getLine(leftNearest.row).charAt(leftNearest.column)]
-            rightBracket = session.getLine(rightNearest.row).charAt(rightNearest.column)
+        closingBrackets = 
+            ")": "("
+            "]": "["
+            "}": "{"
+        openingBrackets = 
+            "(": ")"
+            "[": "]"
+            "{": "}"
+
+        # Next two 'if' needs to avoid conflict with the standard highlight
```

Comment:
I've seen that bug, but I asked if you're sure that you feed `findOpeningBracket` with corret arguments. It starts searching for the bracket from some position: do you give it position before the cursor or at the cursor? If you have {...}_ and you call `findOpeningBracket` with position, corresponding  to } then you get one result, while the same call with position of } plus 1 gives another result.